datetime/datetime_test.go

```diff
@@ -0,0 +1,543 @@
+package datetime_test
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/tarantool/go-tarantool"
+	"github.com/tarantool/go-tarantool/test_helpers"
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+var (
+	MinTime = time.Unix(0, 0)
+	MaxTime = time.Unix(1<<63-1, 999999999)
+)
+
+// There is no way to skip tests in testing.M,
+// so we use this variable to pass info
+// to each testing.T that it should skip.
+var isDatetimeSupported = false
+
+var server = "127.0.0.1:3013"
+var opts = Opts{
+	Timeout: 500 * time.Millisecond,
+	User:    "test",
+	Pass:    "test",
+}
+
+var space = "testDatetime_1"
+var index = "primary"
+
+func connectWithValidation(t *testing.T) *Connection {
+	conn, err := Connect(server, opts)
```

Comment:
It will be better to mark the function as a [test helper](https://pkg.go.dev/testing#T.Helper)
```suggestion
func connectWithValidation(t *testing.T) *Connection {
        t.Helper()
	conn, err := Connect(server, opts)
```

Reply:
Done.

```diff
--- a/datetime/datetime_test.go
+++ b/datetime/datetime_test.go
@@ -33,6 +33,8 @@ var space = "testDatetime_1"
 var index = "primary"
 
 func connectWithValidation(t *testing.T) *Connection {
+       t.Helper()
+
        conn, err := Connect(server, opts)
        if err != nil {
                t.Fatalf("Failed to connect: %s", err.Error())
```